uaclient/entitlements/fips.py

```diff
@@ -226,12 +231,13 @@ def _replace_metapackage_on_cloud_instance(
         if cloud_id is None:
             cloud_id = ""
 
-        cloud_match = re.match(r"^(?P<cloud>(azure|aws)).*", cloud_id)
+        cloud_match = re.match(r"^(?P<cloud>(azure|aws|gce)).*", cloud_id)
         cloud_id = cloud_match.group("cloud") if cloud_match else ""
 
-        if cloud_id not in ("azure", "aws"):
+        if cloud_id not in ("azure", "aws", "gce"):
             return packages
 
+        cloud_id = "gcp" if cloud_id == "gce" else cloud_id
```

Comment:
Let's use our mapping from uaclient.identity.DATASOURCE_TO_CLOUD_ID
```suggestion
        cloud_id = "CLOUD_TYPE_TO_TITLE.get(cloud_id, cloud_id)
```

Reply:
@blackboxsw the problem with this solution is that the `CLOUD_TYPE_TO_TITLE` is not a proper translation for the other clouds, for example, we would fail to get the correct metapackage if we were on an AWS gov cloud

Reply:
great point.

Reply:
Anyhow, not a blocker, because it is a one-off specifically for gce anyway. No need to broaden that net at the moment.